### PR TITLE
fix(Router): update Router interface to SteamOS3.4 and add Navigation

### PR DESCRIPTION
**Pull request by beebls**
Navigation is an aggregator for the depths of hell that Valve has moved the Router functions to.

### CODE REVIEW

**Review by AAGaming00 — APPROVED** [review]

**Comment by AAGaming00** [comment]
Going to test this tomorrow.

**Review by AAGaming00 — CHANGES_REQUESTED** [review]
While this PR functions, it removes lots of functionality because the typings for these methods arguments are missing. While the devtools shows them as having no args, you can see that they use the (deprecated) `arguments` variable, so the arguments are in fact still used and working. Please re-add them.

File: src/deck-components/Router.tsx [hunk 1]
```diff
@@ -119,7 +119,7 @@ export interface Navigation {
   NavigateToSteamWeb(url: string): void;
   NavigateToWebRoute(unknown?: any, unknown2?: any): void;
   OpenSideMenu(sideMenu: SideMenu): void;
-  OpenQuickAccessMenu(): void;
+  OpenQuickAccessMenu(quickAccessTab?: QuickAccessTab): void;
```
Comment (AAGaming00) [inline src/deck-components/Router.tsx:122]: forgot to add this on the other interface

**Review by AAGaming00 — CHANGES_REQUESTED** [review]

File: src/deck-components/Router.tsx (hunk 1 above)
Comment (beebls) [reply src/deck-components/Router.tsx:122]: whoops...

Fixed now!

**Pull request merged**